cg/store/models.py

```diff
@@ -1089,7 +1089,9 @@ class PacBioSequencingRun(InstrumentRun):
     id: Mapped[int] = mapped_column(ForeignKey("instrument_run.id"), primary_key=True)
     well: Mapped[Str32]
     plate: Mapped[int]
-    movie_time_hours: Mapped[int]
+    run_started_at: Mapped[datetime | None]
+    run_completed_at: Mapped[datetime | None]
```

Comment:
The table is named run, so to starting the column name with run is unnecessary.

Reply:
done